minikube/bootstrap.sh

```diff
@@ -91,6 +102,79 @@ upgradeHomebrewPackages() {
   done
 }
 
+# upgradeChocoPackages upgrades required Chocolatey packages to latest version.
+upgradeChocoPackages() {
+  runIfNot "choco list --local-only | grep 'minikube'" \
+    choco install minikube
+  if [ ! $(choco outdated | grep 'minikube') ]; then
+    runCmd \
+      choco upgrade minikube
+  fi
+  for pkg in kubernetes-cli kubernetes-helm; do
+    if [ ! $(choco list --local-only | grep $pkg | awk '{ print $1}') ]; then
+      runCmd \
+        choco install $pkg
```

Comment:
@targs08 make sure that default `choco install/upgrade` behavior has no interaction which requires user prompting, because some package managers does have (`yum`/`apt-get`) and their use requires additional flag to disable interaction.

Reply:
@tyranron, yes with flag `-y, --yes, --confirm` will be better. [More info](https://github.com/chocolatey/choco/wiki/CommandsInstall#options-and-switches)